modules/oneVideoBidAdapter.js

```diff
@@ -214,6 +214,18 @@ function getRequestData(bid, consentData, bidRequest) {
     if (bid.params.video.placement) {
       bidData.imp[0].banner.placement = bid.params.video.placement
     }
+    if (bid.params.video.maxduration) {
+      if (!bidData.imp[0].banner.ext) {
+        bidData.imp[0].banner.ext = {}
+      }
```

Comment:
If you want, you could reduce amount of code by replacing these 3 lines with:

```
bidData.imp[0].banner.ext = bidData.imp[0].banner.ext || {}
```

You could also apply this logic for the `if` condition for `minduration`.